genestack_client/genestack_exceptions.py

```diff
@@ -59,6 +61,33 @@ def __str__(self):
         return message
 
 
+class GenestackConnectionException(urllib2.URLError):
```

Comment:
Having both `GenestackConnectionException` **and** `GenestackConnectionError` is awful, because they are nearly impossible to distinguish.

Reply:
I suggest `GenestackResponseError`.